### ISSUE TITLE
[VR] Splats incorrectly rotate with head tilt

### ISSUE THREAD

**Issue by HuayuSun** [aras-p/UnityGaussianSplatting#148]
GS version: newest 1.0
unity: 2022.3.51f1
platform:Andriod, vulkan
pipeline: BIRP

To reproduce the issue, create an unity scene with a gaussian splat mesh  and XR Interaction Setup Camera in it. Build and open it in HMD(PICO4 in my case) and lean your ear to your shoulder.
This will create the effect of the splats rotating along with the roll of your head, becoming completely misaligned with how they should be viewed, while other meshes created by unity (such as cube or sphere) dont have this problem.

Long story short, I am tilting my head to the left and right (touching my ear to my shoulder). The background plane in the scene rotates fine. But the splats are rotating with my head.

I tried the same build setting in windows DX12 platform, this issue won't show up.

Would love some help.

**Comment by aras-p** [comment]
I have exactly zero VR devices around me, so the fix would have to be contributed by someone else.

**Comment by HuayuSun** [comment]
Switch to URP pipeline and the problem solved.

**Comment by rajgaurav12345** [comment]
> Switch to URP pipeline and the problem solved.

Will it work on Standalone......... am unable to setup It for standalone builds, I need help

**Comment by guozechao** [comment]
请问 在PICO4中实现3dgs 卡顿吗？

**Comment by HuayuSun** [comment]
看你素材高斯数量，五万级差不多能到70帧

________________________________
发件人: guozechao ***@***.***>
发送时间: 2025年1月2日 9:39
收件人: aras-p/UnityGaussianSplatting ***@***.***>
抄送: MelonBear ***@***.***>; State change ***@***.***>
主题: Re: [aras-p/UnityGaussianSplatting] [VR] Splats incorrectly rotate with head tilt (Issue #148)


请问 在PICO4中实现3dgs 卡顿吗？

―

**Comment by guozechao** [comment]
请问下那这个是硬件的限制吗？还是其他问题？

**Comment by aras-p** [comment]
@guozechao @HuayuSun I don't know what you're talking about here, but it does not sound like it is related to this github issue.

**Comment by crazyzyz** [comment]
> Switch to URP pipeline and the problem solved.

Hello, I also encountered this problem. When using pico4u and the latest version of git project, the pico headset moved, and the 3dgs also moved. My current project is URP, and this problem also occurred. Looking forward to your reply!

![Image](https://github.com/user-attachments/assets/d843d0cc-58f4-41fa-be16-b8ed5ece1863)
